docs/src/Provider.doc.js

```diff
@@ -93,59 +94,68 @@ function Example(props) {
+      title="Custom navigation context"
+      description={`
+        These examples illustrate a custom implementation of \`onNavigation\` context to control the link functionality of Gestalt components externally.
+
+        This example has 4 relevant parts: a Provider, an \`onNavigation\` high-order function, consumer components, and \`onNavigationOptions\` props.
+
+        The top Provider passes the custom \`onNavigation\` function to consumer components. Then, \`onNavigation\` returns a function that gets called during the \`onClick\` event handler.
+
+        The \`onNavigation\` function can contain complex logic including React hooks to perform side effects. In this case, \`onNavigation\` is used to disable the default link, show an alert message, and open a different URL in a new window.
+
+        Finally, the \`onNavigationOptions\` prop provides a flexible API. The \`onNavigationOptions\` is passed as an argument to the \`onNavigation\` function providing external control to the logic inside the function. In this case, \`onNavigationOptions\` toggles a \`navigationMode\` key between \`default\` and \`navigation\` values. Then, inside the \`onNavigation\` function, the \`navigationMode\` value toggles between the default link behaviour and the custom navigation context one. Other uses could be accessing \`event.stopPropagation\`.
+      `}
     >
       <MainSection.Card
+        title="Examples above from left to right: (1) Link, (2) Button, (3) IconButton, (4) TapArea"
         cardSize="lg"
         defaultCode={`
 function OnNavigation() {
-  const [clientOnNavigationMode, setClientOnNavigationMode] = React.useState(true);
+  const [clientOnNavigationMode, setClientOnNavigationMode] = React.useState('link');
 
   const onNavigation = ({ href, onNavigationOptions }) => {
 
     const onNavigationClick = ({ event }) => {
       event.nativeEvent.preventDefault();
       // eslint-disable-next-line no-alert
-      alert("Disabled link. Opening help.pinterest.com instead.");
-      window.open("https://help.pinterest.com", "_blank");
+      alert('Disabled link. Opening help.pinterest.com instead.');
+      window.open('https://help.pinterest.com', '_blank');
     }
 
-    return onNavigationOptions && onNavigationOptions.navigationMode === "client"
+    return onNavigationOptions && onNavigationOptions.navigationMode === 'navigation'
```

Comment:
Think these should each be 'onNavigation' (or the radios pass in 'navigation'

Reply:
DONE